src/Components/InputTodo.js

```diff
@@ -0,0 +1,89 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { runToast } from './toast/Toast';
+
+const InputTodo = ({ list, setList }) => {
+    const [form, setForm] = useState('');
+    const handleChange = (e) => {
+        setForm(e.target.value);
+    };
+    const handleInput = () => {
+        const createItem = () => {
+            setList([
+                ...list,
+                {
+                    id: uuidv4(),
+                    content: form,
+                    done: false,
+                },
+            ]);
+        };
+
+        form // 입력한 값이 있고,
+            ? !list.find((item) => item.content === form) //중복되는 값이 없을 때
+                ? createItem()
+                : runToast('동일한 내용으론 작성할 수 없어요!')
+            : runToast('할 일을 입력해주세요!');
```

Comment:
이 부분은 ternary operator를 사용하시는 것보다는 그냥 if문을 사용하시는 게 가독성이 더 좋을 것 같습니다

```suggestion
        if (form) {
           if (list.find((item) => item.content === form)) {
             RunToast('동일한 내용으론 작성할 수 없어요!');
           } else {
             createItem();
           }
         } else {
           RunToast('할 일을 입력해주세요!');
    }
```

Reply:
확실히 아래가 보기 더 편하네요..!!